File: hackerrank.com/problem-solving/grading.scala
// https://www.hackerrank.com/challenges/grading/problem
import scala.io.StdIn._

object Solution {

  def gradingStudents(grades: Array[Int]): Array[Int] = {
    grades.map(i => if (i < 38 || i % 5 < 3) i else i + 5 - i % 5)
  }

  def main(args: Array[String]) {
    val n = readInt
    val grades = for (i <- 0 until n) yield readInt
    val newGrades = gradingStudents(grades.toArray)
    println(newGrades.mkString("\n"))
  }
}

File: hackerrank.com/problem-solving/grading.js
// https://www.hackerrank.com/challenges/grading/problem

function gradingStudents(grades) {
    return grades.map(x => x < 38 ? x : (x % 5 > 2 ? x + 5 - x % 5 : x));
} 


function main(inputData) {
    const lines = inputData.split("\n");
    const n = parseInt(lines[0])
    const ar = lines.slice(1, n + 1).map(x => parseInt(x))
    console.log(gradingStudents(ar).join('\n'))
}

process.stdin.resume();
process.stdin.setEncoding("ascii");
let _input = '';
process.stdin.on("data", input => {
    _input += input;
});

process.stdin.on("end", () => {
    main(_input);
});